server/internal/app/grpc.go

```
@@ -0,0 +1,171 @@
+		if op := adapter.AcOperator(ctx); op != nil && ar != nil {
+			// apply filters to repos
+			ar = ar.Filtered(accountrepo.WorkspaceFilterFromOperator(op))
+		}
+
+		uc := interactor.NewContainer(r, g, ar, ag, interactor.ContainerConfig{})
+		ctx = adapter.AttachUsecases(ctx, &uc)
+		return handler(ctx, req)
+	}
+}
+
+func tokenFromGrpcMetadata(md metadata.MD) string {
```

golangci-lint (1.64.8)

func `tokenFromGrpcMetadata` is unused

(unused)